pandas/core/frame.py

```diff
@@ -3924,30 +3925,45 @@ def rename(self, *args, **kwargs):
 
         Parameters
         ----------
-        mapper, index, columns : dict-like or function, optional
-            dict-like or functions transformations to apply to
+        mapper : dict-like or function
```

Comment:
OK as now I think but generally unrelated changes like this are better done in a separate PR

Reply:
I understand, I just figured it might be nice to reduce the technical debt of this method, as I saw that this docstring contained numerous errors. Will keep it in mind for next time!